docs/AddVisitSequenceDiagram.puml

```diff
@@ -0,0 +1,75 @@
+@startuml
+'https://plantuml.com/sequence-diagram
+
+actor Doctor
+
+Doctor -> ":Main" ++ : addVisit ic/S9870789B v/29-10-2024 10:00 d/Gastric m/Gaviscon, Paracetamol
+
+":Main" -> ":Scanner"**: new Scanner()
+activate ":Scanner"
+":Main" <-- ":Scanner": in:Scanner
+deactivate ":Scanner"
+
+":Main" -> ":Records"**: new Records()
+activate ":Records"
+":Main" <-- ":Records": records:Records
+deactivate ":Records"
+
+":Main" -> ":AppointmentRecord"**: new AppointmentRecord()
+activate ":AppointmentRecord"
+":Main" <-- ":AppointmentRecord": appointmentRecord:AppointmentRecord
+deactivate ":AppointmentRecord"
+
+participant ":FileHandler" as FileHandler <<class>>
+":Main" -> FileHandler ++: initFile(records)
+return
+
+":Main" -> FileHandler ++: initFile(appointmentRecord)
+return
```

Comment:
There seems to be an overlap with the MainSequenceDiagram.puml. Moving this part to a separate diagram and just referencing it would improve readability, would you agree?